### DISCUSSION TITLE
No schortcuts / launchers after installation. Launchers are missing!

### DISCUSSION THREAD

**Discussion by Ng1n3eR** [moraroy/NonSteamLaunchers-On-Steam-Deck#498] [Q&A]
After installation there are no schortcuts of apps and theirs folders are empty. I checked everything to be installed and only youtube and disney+ are working but 0 launchers - even ea app have title repair instead of normal and its not launching too. Uplay launcher is missing the same for GOG and others. please help me. below my log from install. Tried thousand times installing with no luck and after fresh start nothing happens too.

[NonSteamLaunchers-install.log](https://github.com/user-attachments/files/17781385/NonSteamLaunchers-install.log)

**Comment by frankmy28** [comment]
I have the same problem when trying to install GOG using the Decky plugin, Epic Games worked for me

**Comment by 5uck3rpunch** [comment]
This issue still persists.  Is the dev still active with this plugin?

**Reply by moraroy** [reply]
@5uck3rpunch what seems to be your issue? Please give more details... are you isng thr desktop or the plugin ?

**Reply by 5uck3rpunch** [reply]
I am using the plugin on Bazzite.  The plugin in Decky goes through the process of installing all of the store apps, but then they are nowhere to be found.  I'm at work & don't have the device with me to pull the logs.  I have uninstalled & reinstalled the plugin.

**Reply by moraroy** [reply]
@5uck3rpunch have you updated the Decky Plugin version to the latest? Run NSLPlugin.desktop this will give you the most up to date decky plugin. Once you have confirmed that please show me the logs so I can help further. I'm sure it's a small issue as I've done alot of updates and fixes recently :)

**Reply by 5uck3rpunch** [reply]
I will do this when I get home from work later.  Thank you again for all of your hard work (and the Bible verses!)!  Also, thanks for responding so quickly here & Reddit.  You rock!

**Reply by 5uck3rpunch** [reply]
> @5uck3rpunch have you updated the Decky Plugin version to the latest? Run NSLPlugin.desktop this will give you the most up to date decky plugin. Once you have confirmed that please show me the logs so I can help further. I'm sure it's a small issue as I've done alot of updates and fixes recently :)

@moraroy Where do I find the logs for the Decky plugin version please?

**Reply by moraroy** [reply]
@5uck3rpunch home/deck/Downloads and home/deck/homebrew/logs/NonSteamLaunchers/

**Comment by 5uck3rpunch** [comment]
I attached the logs here

**Comment by 5uck3rpunch** [comment]
adjusted the screen resolution to get to the Ok button & installed a few
stores and then power cyled the unit & the stores are still missing from
the Steam library.

**Reply by moraroy** [reply]
@5uck3rpunch message me on discord @moraroy

**Reply by 5uck3rpunch** [reply]
I got it working.  5th time is a charm.  I had to remove the NonSteamLaunchers plugin from Ducky in game mode.  I then went into desktop mode & opened the latest version of NonSteamLaunchers Desktop & clicked on Start Fresh.  I then powered the device off and waited & then back on.  Went back into desktop mode & ran NonSteamLaunchers Desktop again. Finally, during the update process after clicking OK & leaving the Custom Web Links box blank, the chosen store apps installed.  I have done this 5 or 6 times & this time it finally worked & I now see all of the stores it installed in Steam under the Other category.
Trying to install just from the Decky plugin did not work for me & I tried that about 10 times before the multiple tries from the desktop version.  I have a solid & fast (1G fiber) internet connection.
Maybe I had these issue due to Bazzite being installed on an AyaNeo 2?  I dunno.  I would suggest to add the ability to resize the window on the desktop version.  It only resizes horizontally & not vertically.  I think it's a must on handheld devices.  The only way around getting to the OK button on the menu to start the install was to jack up the screen resolution, which makes it almost impossible to read or to get the correct button using the touch screen.
Keep up the great work & thanks.  Your app is way better than the other limited apps like Heroic.  God Bless & stay safe Sir.

**Reply by moraroy** [reply]
@5uck3rpunch I pushed alot of fixes about an hour ago that actually probably helped in your situation! I'm glad the dssktop version is finally working. Be sure to always keep the Plugin version up to date! I will continue to check and see what the issue is now on the plugin version. I'll also take a look at that resizing the ze it window, problem is I have too many buttons now in just one window so I'll have to see what I can do and try to change it vertically, God Bless you too! I'll continue to improve the script slowly but surely! Thank you for being patient<3

**Reply by 5uck3rpunch** [reply]
I would suggest to maybe put the non-essential buttons maybe one button that brings up a sub menu where you can access all of them.  Or maybe do 2 less wide rows of buttons. Good luck.

**Reply by moraroy** [reply]
@5uck3rpunch try the .desktop file , I just now added a resolution check let me know I that size is better

**Reply by 5uck3rpunch** [reply]
I will test it when I get home from work.  Thanks.
Edit: Is that in the 4.1.6 version?

**Reply by 5uck3rpunch** [reply]
Yesterday's latest update will not install the humble bundle collection with either the desktop or the plug-in version. I uninstalled the plug-in and the desktop version reinstall the latest again and it will not install humble bundle no matter which way I do it

**Reply by moraroy** [reply]
@5uck3rpunch dang ok will take a look :/

**Reply by 5uck3rpunch** [reply]
[NonSteamLaunchers-install.log](https://github.com/user-attachments/files/20061055/NonSteamLaunchers-install.log)
